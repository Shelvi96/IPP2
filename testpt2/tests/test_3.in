NEW id1
001 > 123
? 123
DEL id1
NEW id2
000000000000000000000000000000001 > 123456789012345678901234567890
? 123456789012345678901234567890
DEL id2
NEW id3
456666666666666666666666666666666666666666666666666666666666 > 001
? 001
DEL id3
NEW id4
456 > 000000000000000000000000000000001
? 000000000000000000000000000000001
DEL id4
