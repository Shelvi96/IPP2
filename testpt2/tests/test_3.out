001
123
000000000000000000000000000000001
123456789012345678901234567890
001
456666666666666666666666666666666666666666666666666666666666
000000000000000000000000000000001
456
